primitives/crypto-ec-utils/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[package]
+name = "sp-crypto-ec-utils"
+version = "0.4.0"
```

Comment:
Nitpick. Why not "0.1.0"?

Reply:
Well, yes. The reason is that it provides the Arkworks host functions for Arkworks version `0.4.0`. Would you recommend to pick `0.1.0` instead and write somewhere else that it provides host functions for Arkworks `0.4.0`?